UM/CentralFileStorage.py

```diff
@@ -34,59 +35,62 @@ class CentralFileStorage:
     """
 
     @classmethod
-    def store(cls, file_path: str, file_id: str, version: Version = Version("1.0.0")) -> None:
+    def store(cls, path: str, path_id: str, version: Version = Version("1.0.0")) -> None:
         """
         Store a new file into the central file storage. This file will get moved to a storage location that is not
         specific to this version of the application.
 
         If the file already exists, this will check if it's the same file. If the file is not the same, it raises a
         `FileExistsError`. If the file is the same, no error is raised and the file to store is simply deleted. It is a
         duplicate of the file already stored.
-        :param file_path: The path to the file to store in the central file storage.
-        :param file_id: A name for the file to store.
+        :param path: The path to the file to store in the central file storage.
```

Comment:
That's not quite correct. You can store a file _or_ a folder now right? (make sure to update that in all other params as well)

Reply:
Good catch, fixed.